pocs/dome/__init__.py

```diff
@@ -1,6 +1,6 @@
 from abc import ABCMeta, abstractmethod, abstractproperty
 
-import pocs
+from pocs.base import PanBase
```

Comment:
Why change this import? I think it undoes a change I made earlier, IIRC.

Reply:
This was failing with just `import pocs`. We don't have anything in the `pocs/__init__.py` file so it wasn't pulling it in.

Could change to `import pocs.base` and then leave `pocs.base.PanBase` but I just imported this specifically.

Reply:
Looks like you changed it back in #418. I think things are much cleaner now.

Reply:
@jamessynge did you have concerns about this affecting the astrohaven domes?

Reply:
Not a concern about the dome, just that we will benefit from a consistent style ... especially if it works well! ;-)